forc-test/src/lib.rs

```diff
@@ -208,10 +228,12 @@ fn run_tests(built: BuiltTests) -> anyhow::Result<Tested> {
         })
         .collect::<anyhow::Result<_>>()?;
 
-    let built = built_pkg;
-    let tested_pkg = TestedPackage { built, tests };
-    let tested = Tested::Package(Box::new(tested_pkg));
-    Ok(tested)
+    let tested_pkg = TestedPackage {
+        built: Box::new(built_pkg.clone()),
```

Comment:
Rather than cloning the whole package here, it looks like we can take `built_pkg` by value in the function input rather than by reference, i.e. `built_pkg: BuiltPackage`.